docs/updating-yarn.md

```diff
@@ -0,0 +1,32 @@
+# Updating Yarn
+
+## Overview
+
+To keep workflows and tooling easy to maintain, we use a unique yarn version across all workspaces. This document outlines the process for updating the yarn version across all workspaces.
+
+## Process
+
+> [!NOTE] this is the process used from version 3.x to 4.x, future version could imply a different process.
+
+### 1. Update Yarn Version
+
+update the yarn version runing this command on the root of the repository:
+
+```bash
+yarn set version latest # or a specific version if is needed
+```
+
+### 2. Update Workspaces
+
+Bumping from version 3 to 4 implies changes on all the workspaces `yarn.lock` files, to update all the following command can be used:
+
+```bash
+chmod +x ./scripts/yarn-install-all.sh
+./scripts/yarn-install-all.sh
+```
+
+What this command does is to iterate over all the workspaces and run `yarn install --mode=update-lockfile` on each one. The `--mode=update-lockfile` flag is used to update the `yarn.lock` file only saving time form yarn link and other steps.
+
```

Comment:
Need to change this as `--mode=update-lockfile` on this version a full install was needed instead to get the correct checksums 
ended up running `./scripts/run-in-all.sh "yarn install && yarn dedupe --mode=update-lockfile && rm -rf node_modules"` doing full instal  and removing the node_modules on each one to not kill my disk space